### ISSUE TITLE
Tags not ignored with karma-jasmine 3.1

### ISSUE THREAD

**Issue by rajsite** [mnasyrov/karma-jasmine-spec-tags#7]
While updating my devDependencies to latest I found that if I update karma-jasmine to latest (v3.1.0) that my `--skipTags` are no longer functional.

Staying at karma-jasmine 2.0.1 or upgrading to karma-jasmine 3.0.3 seem to work with karma-jasmine-spec-tags but when using karma-jasmine 3.1.0 my skipped tags are no longer skipped.

**Comment by mnasyrov** [comment]
As I see the latest karma-jasmine changed a way to configure jasmine environment. It takes a time to fix.

**Comment by lgrignon** [comment]
Hello, 
Is there any progress on this please? :)

(In the mean time, I rollbacked my karma-jasmine to 3.0.3 and it works. Thank you for this nice plugin!)

**Comment by mnasyrov** [comment]
@rajsite @lgrignon 
I opened PR #9 with a fix. Can you pull this branch and test in your environments?
```bash
npm install https://github.com/mnasyrov/karma-jasmine-spec-tags/tarball/fix/specFilter-injection
```

The fix should work with all karma-jasmine versions, so it will be a minor release.

**Comment by lgrignon** [comment]
It works perfectly, thanks a lot! :)

**Comment by lgrignon** [comment]
Could you please notify here when released? Thanks again

**Comment by mnasyrov** [comment]
Published as `karma-jasmine-spec-tags@1.3.0`

**Comment by lgrignon** [comment]
thanks :)
